compiler/cpp/src/thrift/generate/t_java_generator.cc

```diff
@@ -2108,8 +2117,8 @@ void t_java_generator::generate_java_struct_compare_to(ostream& out, t_struct* t
  *
  * @param tstruct The struct definition
  */
-void t_java_generator::generate_java_struct_reader(ostream& out, t_struct* tstruct) {
-  (void)tstruct;
+void t_java_generator::generate_java_struct_reader(ostream& out, t_struct* /*tstruct*/) {
```

Comment:
You commented out the variable name, but left the variable's type. I'm mostly a Java dev. Is that a C++ quirk that's allowed for unused variables?

Reply:
both are valid ways to suppress warnings about unused variables (in this case an error because `-Werror` flag is present).

https://stackoverflow.com/questions/308277/what-are-the-consequences-of-ignoring-warning-unused-parameter/308286#308286

i'd prefer the commenting out way because an extra line with `(void)` is a bit arcane.

Reply:
That's fine... I just didn't know you could comment out the variable name, but leave the type in place. You definitely can't do that in Java, but it does make sense... you're keeping the same signature just not using the variable. :smiley_cat: